source/hic_hal/freescale/kl27z/DAP_config.h

```diff
@@ -68,7 +68,7 @@ Provides definitions about:
 /// Default communication speed on the Debug Access Port for SWD and JTAG mode.
 /// Used to initialize the default SWD/JTAG clock frequency.
 /// The command \ref DAP_SWJ_Clock can be used to overwrite this default setting.
-#define DAP_DEFAULT_SWJ_CLOCK   5000000         ///< Default SWD/JTAG clock frequency in Hz.
+#define DAP_DEFAULT_SWJ_CLOCK   8000000         ///< Default SWD/JTAG clock frequency in Hz.
```

Comment:
I'm wondering if making the default clock 8 MHz is ok. On the micro:bit you have short SWD traces and a known design, so it's always fine. But for other boards it may not be reliable.

If we do decide to change it, it should be configurable with a macro. So this line would need to be wrapped with `#if !defined(DAP_DEFAULT_SWJ_CLOCK)`.

Reply:
Made this change board specific in the .yaml file.